src/filesplugin.js

```diff
@@ -15,11 +15,32 @@ import { generateUrl } from '@nextcloud/router'
 import { showSuccess, showError } from '@nextcloud/dialogs'
 import { translate as t, translatePlural as n } from '@nextcloud/l10n'
 import { oauthConnect, oauthConnectConfirmDialog, gotoSettingsConfirmDialog, SEND_TYPE } from './utils.js'
+import {
+	registerFileAction, Permission, FileAction,
+	davGetClient, davGetDefaultPropfind, davResultToNode, davRootPath,
+} from '@nextcloud/files'
+import { subscribe } from '@nextcloud/event-bus'
+import MattermostIcon from '../img/app-dark.svg?raw'
```

Comment:
`import MattermostIcon from '../img/app-dark.svg'`

Reply:
This does not work, Webpack encodes it in base64 and we need the raw one.
Any idea how to get rid of this warning? `Unable to resolve path to module '../img/app-dark.svg?raw'`
(apart from ignoring `import/no-unresolved` in the eslint config :grin: )

Reply:
ohh.

> apart from ignoring import/no-unresolved in the eslint config 😁

haha sure

This does the trick without any warnings (in webpack.js):
```
webpackConfig.module.rules.push({
	test: /\.svg$/i,
	type: 'asset/source',
})
```
^^ replace `resourceQuery: /raw/` and do the import normally.

Reply:
:blue_heart:  Thanks, done.